templates_apprunner/templates_cspm_eventbridge/FullInstall.yaml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: IAM Role for Agentless
```

Comment:
This template does more than just IAM role

Reply:
added correct description.